apps/dashboard/src/routes/+page.svelte

```diff
@@ -89,11 +89,13 @@
 			};
 			const { error: urlError } = await data.supabase.from('urls').insert(newURL);
 			if (urlError) {
-				var msg =
-					urlError.code == '23505'
-						? `URL Prefix with the domain "${subdomain + domain}" already exists!`
-						: toastError.message;
-				toastStore.trigger({ ...toastError, message: msg });
+				// Attempt to revert the previous add domain call.
```

Comment:
I was thinking we do the revert in the add-domain api-call itself

Reply:
unless I'm missing something, but how would we know to revert?

If the vercel API call goes through, and the database call doesn't afterwards, then we revert the vercel API call. We need to know the status of the database call.

Are you suggesting that we move the database call inside of the vercel API call?